sakura-argo/app/kustomization.yaml

```
@@ -0,0 +1,11 @@
+helmCharts:
+  - repo: https://argoproj.github.io/argo-helm
+    name: argo-workflows
+    # renovate:general datasource=helm depName=argo-workflows registryUrl=https://argoproj.github.io/argo-helm
+    version: "0.45.22"
+    includeCRDs: true
+    namespace: sakura-argo
+    releaseName: sakura-argo
+    valuesFile: values.yaml
```

ここは共通部分を排除するために、既存の`argo`からvalueを読むようにしたいです
```suggestion
    valuesFile: ../../argo/app/values.yaml
```